package.json

```diff
@@ -6,23 +6,40 @@
   "author": {
     "name": "Consumer Financial Protection Bureau",
     "email": "tech@cfpb.gov"  ,
-        "url": "http://consumerfinance.gov" 
+        "url": "http://consumerfinance.gov"
   },
   "license": "CC0",
   "keywords": [
     "design-manual"
   ],
+  "dependencies": {
+    "cf-buttons": "^3.0.3",
```

Comment:
Ah right, the bot publishes them all individually. I was thinking this pointed back to the original repos